weblab/repocache/tests/test_populate.py

```diff
@@ -1,28 +1,52 @@
 import pytest
-
+import os.path
+from pathlib import Path
 from core import recipes
+from accounts.models import User
 from repocache.populate import populate_entity_cache
 
 
 @pytest.mark.django_db
 class TestPopulate:
-    def test_populate(self, model_with_version):
+    def test_populate(self, model_with_version, helpers):
         model_with_version.repo.tag('v1')
-
         populate_entity_cache(model_with_version)
-
         cached = model_with_version.cachedentity
-
         assert cached is not None
-        latest = model_with_version.repo.latest_commit
 
-        version = cached.versions.get()
-        assert version.sha == latest.sha
-        assert version.message == latest.message
-        assert version.timestamp == latest.timestamp
+        latest = model_with_version.repo.latest_commit
+        version1 = cached.versions.get()
 
+        assert version1.sha == latest.sha
+        assert version1.message == latest.message
+        assert version1.timestamp == latest.timestamp
         assert cached.tags.get().tag == 'v1'
 
+        # create second version with master_filename
+        model_with_version.repo.tag('v2')
+
+        # set up required files and manifest
+        path = os.path.join(model_with_version.repo._root, 'file.cellml')
```

Comment:
Rather than adding an extra file in the second commit, you can just add the manifest file. The initial commit includes a file `file1.txt` (see conftest.py `Helpers.add_version`) so you can just do
```python
model_with_version.repo.generate_manifest(master_filename='file1.txt')
commit = model_with_version.repo.commit('second commit', latest.author)
```

Reply:
done, didn't seem to be able to use latest.author as it complains about not having a fullname so constructed a new User object for the purpose